packages/react-magma-dom/src/components/Input/Input.test.js

```diff
@@ -296,13 +296,15 @@ describe('Input', () => {
 
   it('should clear the input when the clear input button is clicked', () => {
     const onClear = jest.fn();
+    const onDateChange = jest.fn();
     const labelText = 'Input Label';
     const value = 'Test Value';
     const { getByTestId, getByLabelText } = render(
       <Input
         labelText={labelText}
         value={value}
         onClear={onClear}
+        onDateChange={onDateChange}
```

Comment:
This should be an optional prop-- most inputs won't have dates, so we shouldn't expect adopters to use this prop

Reply:
fixed optional prop using